### DISCUSSION TITLE
firestick 4k max issues

### DISCUSSION THREAD

**Discussion by kiasho** [awawa-dev/HyperHDR#633] [General]
I have a firestick 4k max connected directly to my usb grabber and the led's work great the issue im having is momentary black screens on the tv which doesnt appear to happen when the firestick is plugged directly into the tv with no ambilight.
Has anyone else had this issue.

**Comment by AstaRom** [comment]
In my setup, I've been using FTV4K Max for 1.5 years now.
I do not observe any bands both when connecting directly to the capture card, and when connected via an HDMI splitter (Chdmi-8M or Ezcoo).
- If you turn off the capture card and leave only the splitter and the Amazon flash drive, does the problem remain?
- Are you using a quality HDMI cable from the splitter to the TV.

**Reply by kiasho** [reply]
I fixed the intermittent black screen issue by using a different hdmi lead from card to tv but now i have another issue when i plug my pc hdmi into the card it changes the screen resolution and everything gets really small on windows

**Reply by AstaRom** [reply]
Well, if you connected the TV as a monitor, then this is natural, since it has a resolution of 4k. You can set the item scale (175-225%) in the display options in Windows settings.
